phoenicis-apps/src/main/java/org/phoenicis/apps/BackgroundRepository.java

```diff
@@ -59,4 +59,19 @@ public void getScript(List<String> path, Consumer<ScriptDTO> callback, Consumer<
     public void setFilter(CombinedAppsFilter filter) {
         delegatedRepository.setFilter(filter);
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Don't forget the {} symbols, and please use EqualsBuilder and HashcodeBuilder as possible